### PR TITLE
Add ECK How to fix a corrupt index exception blog

### PR DESCRIPTION
**Pull request by rubyrat**
@maniankara can you check this looks ok for you when you get a moment. 
I created another tag on the blog page for 'Elastic', any others it should be tagged with? @jlarfors any opinions on the tags?

### CODE REVIEW

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+type: Blog
+title: 'Elastic Cloud Kubernetes: How to fix a corrupt index exception'
```
Comment (jlarfors) [inline src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:3]: Should this actually be "Elastic Cloud on Kubernetes"? Or is "Elastic Cloud Kubernetes" the thing?

https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+subheading: In this article we’re going to see how to fix a corrupt index which was not automatically recovered.
+authors:
+- avijayan
+tags:
+- Elastic
```
Comment (jlarfors) [inline src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:8]: - Kubernetes
- Cloud

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+The tool opens the index and tries to fix it the corrupted parts of the index after getting the permission from the user.
+
+Finally the tool requests the user to `reroute` the index to allocate a primary shard on a node. Sometimes, its needed to get the whole cluster up to `reroute` the index. In that case, you need to run the `reroute` after the node it attached back to the cluster.
+
+### Attach the node back to the cluster
+
+Now that the corruption is fixed, it’s time to start the Elasticsearch node and enable the shard allocations settings  to `all` . The node can be restarted by removing the annotations with the command
+
+```jsx
```
Comment (jlarfors) [inline src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:63]: `bash`, not`jsx`...

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+In order to execute this command in an Elasticsearch node, the Elasticsearch service should not be running. This is to prevent any read/write actions while the shard is being repaired. This is complicated in a container-based environment like Kubernetes because the container exits if the entry point process is stopped, in this case, the Elasticsearch process.
+
+However, Elastic Cloud Kubernetes comes with a way to suspend containers with the help of Kubernetes annotations. When annotated with [`eck.k8s.elastic.co/suspend`](http://eck.k8s.elastic.co/suspend) the container restarts into one of init containers, namely `elastic-internal-suspend` with all the necessary disks mounted. After connecting to that internal container, one can have access to the disk and can run commands.
```
Comment (jlarfors) [inline src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:23]: Another reference to "Elastic Cloud Kubernetes". Should we add the "on" inbetween?

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md (hunk 1 above)
Comment (jlarfors) [reply src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:3]: If it is with "on" remember to update the image for this blog also.

**Review by jlarfors — CHANGES_REQUESTED** [review]
Some questions/comments and tags to answer/fill.

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md (hunk 1 above)
Comment (maniankara) [reply src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:3]: Hi, Its Elastic Cloud on Kubernetes

File: src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md (hunk 4 above)
Comment (maniankara) [reply src/posts/elastic-cloud-kubernetes-how-to-fix-a-corrupt-index-exception.md:23]: Yes, please

**Pull request merged**